hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneManagerLock.java

```diff
@@ -202,6 +196,26 @@ private boolean lock(Resource resource, String resourceName,
     }
   }
 
+  private void updateLockMetrics(Resource resource, String resourceName,
```

Comment:
Please rename the method as its doing actual locking as well as updating metrics.